packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -459,6 +466,7 @@ export type CallControlOptions = (CommonCallControlOptions & {
     participantsButton?: boolean | {
         disabled: boolean;
     };
+    legacyControlBarExperience?: boolean;
```

Comment:
Do we have UI tests for this?

Reply:
No but will add that.

Reply:
Edit: My mistake. All previous tests in CallControls.test.ts use the legacyControlBarExperience so we are still taking snapshots of the old control bar. The new snapshots  with the new control bar in this PR were generated because the tests with legacyControlBarExperience equalling false were added to stable (ie. the conditional compilation statement was removed in the test.)